src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/metadata/doctrine-1.0.xsd

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How about calling this class something like `doctrine-field-descriptors-1.0.xsd`, since we are not redefining doctrine here?